src/tests/ftest/util/dmg_utils.py

```diff
@@ -427,125 +427,61 @@ def pool_create(self, scm_size, uid=None, gid=None, nvme_size=None,
 
         return data
 
-    def pool_query(self, pool):
+    def pool_query(self, pool, use_json=True):
```

Comment:
I would not recommend this approach.  We should eventually be converting all of these `DmgCommand` helper methods to use json output, so why not fully convert this one?

Reply:
Thanks for the review. I'll remove this argument in my next PR.